toolchain/check/cpp/type_mapping.cpp

```diff
@@ -39,16 +42,29 @@ static auto FindIntLiteralBitWidth(Context& context, SemIR::InstId arg_id)
       arg_const_id.is_symbolic()) {
     // TODO: Add tests for these cases.
     return IntId::None;
+    ;
   }
   auto arg = context.insts().GetAs<SemIR::IntValue>(
       context.constant_values().GetInstId(arg_const_id));
-  unsigned arg_non_sign_bits =
-      context.ints().Get(arg.int_id).getSignificantBits() - 1;
+  llvm::APInt arg_val = context.ints().Get(arg.int_id);
+  unsigned arg_non_sign_bits = arg_val.getSignificantBits() - 1;
```

Comment:
```suggestion
  int arg_non_sign_bits = arg_val.getSignificantBits() - 1;
```
nit: I think the google style guide wants use to use `int` here since the value is clearly in range.

Reply:
Done! Thanks.